.github/workflows/test-gb-25.yml

```diff
@@ -86,6 +86,9 @@ jobs:
           # Modify commit Enzyme-JAX commit
           sed -i.bak 's/ENZYMEXLA_COMMIT = ".*"/ENZYMEXLA_COMMIT = "${{ github.sha }}"/' ReactantExtra/WORKSPACE
 
+          # Modify XLA commit
+          # sed -E -i.bak -e 's/(# )?XLA_COMMIT = ".*"/XLA_COMMIT = "0123456789abcdef0123456789abcdef01234567"/' -e 's/(# )?XLA_SHA256 = ""/XLA_SHA256 = ""/' ReactantExtra/WORKSPACE
```

Comment:
Two things:
1) we also need to comment out or delete the load of the xla commit from Jax
2) we should match and hash

Reply:
We should also add a series of targets, like
Xla_hash:
- ""
- "abcd..."

To the github actions yml, and the have it use the hash if non empty

Reply:
Uhm, not sure how you mean exactly.  Do you have an example?

Reply:
Ok, done in [`09919a3` (#1197)](https://github.com/EnzymeAD/Enzyme-JAX/pull/1197/commits/09919a3e2112bc7c7ada855114f55132524e9004)